### PR TITLE
Enable automatic upgrade supports for connectedvmware VM extension.

### PR DESCRIPTION
**Pull request by 93mishra**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repository and upgrade the version in the pull request but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
connectedvmware

**Comment by nascarsayan** [comment]
Can we add unit tests for extension create and extensions?

**Comment by 93mishra** [comment]
> @93mishra, Can we add unit tests for extension create and update?

File: src/connectedvmware/azext_connectedvmware/_params.py [hunk 1]
```diff
@@ -389,6 +389,9 @@ def load_arguments(self, _):
                 'type_', options_list=['--type'], type=str, help='Specify the type of the extension; an example '
                 'is "CustomScriptExtension".')
             c.argument('type_handler_version', type=str, help='Specifies the version of the script handler.')
+            c.argument(
+                'enable_auto_upgrade', arg_type=get_three_state_flag(), help='Indicates whether the extension '
+                'should be automatically upgraded by the platform if there is a newer version available.')
```
Comment (zhoxing-ms) [inline src/connectedvmware/azext_connectedvmware/_params.py:394]:
```suggestion
                'enable_auto_upgrade', arg_type=get_three_state_flag(), help='Indicate whether the extension '
                'should be automatically upgraded by the platform if there is a newer version available.')
```

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**